readme.md

```diff
@@ -0,0 +1 @@
+This project is used for employee_management
```

Comment:
Could you add more description ? @Susmitha27

Reply:
Changes are done!